docs/installation.md

```diff
@@ -58,6 +58,17 @@ npm install @ownclouders/file-picker --save
 yarn add @ownclouders/file-picker
 ```
 
+## Select browser storage
+In order to authorize any request to the ownCloud server, we are storing the Bearer token in a browser storage. By default, it is the session storage. As browsers are adding more strict policies when it comes to blocking third party cookies, your users might experience issues with the token not being saved in the storage. For this reason, it is possible to specify a storage name in the config of File picker. To make it possible to run the File picker inside of an iframe, specify `storage: memory`.
+
+{{< hint danger >}}
+If the memory storage is used, it is not persisted in the session. This leads to users having to authorize again after a refresh has happened.
+{{< /hint >}}
+
+{{< hint info >}}
+Users might still be experiencing issues with the authorization even if memory storage is used. That can happen due to authorization popup needing to trigger a callback in the File picker. To make sure it will work as supposed, be sure to set correct CORS headers.
```

Comment:
```suggestion
Users might still be experiencing issues with the authentication even if the memory storage is used. That can happen due to the authentication popup needing to trigger a callback in the File picker. To make sure it will work as supposed, be sure to set correct CORS headers.
```